lib/rubocop/config_loader.rb

```diff
@@ -165,6 +166,20 @@ def resolve_inheritance(path, hash)
         end
       end
 
+      def resolve_inheritance_from_gems(hash, gems)
+        (gems || {}).each_pair do |gem_name, config_path|
+          hash['inherit_from'] ||= []
+          hash['inherit_from'] << gem_config_path(gem_name, config_path)
```

Comment:
If `hash['inherit_from']` already contains a string, e.g. `.rubocop_todo.yml`, this will concatenate to that string.